plugins/scoreboard.js

```diff
@@ -153,6 +170,7 @@ function displayScoreboard () {
   } else {
     dialog({
       html: html,
+      width: 400,
```

Comment:
This feels very wrong.
Not all displays will handle that.
It's too static to look good at few places (we'd like it to look good everywhere/on every device)

But not sure what to suggest otherwise

Reply:
I have added a commit that uses "auto" instead of a concrete width. That works good in all tested browsers.